misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/OptionalAssertionsSpec.kt

```diff
@@ -27,12 +28,12 @@ abstract class OptionalAssertionsSpec (
         val isEmptyFun = isEmpty.lambda
 
         context("is empty") {
-            it ("does not throw") {
+            it("does not throw") {
                 val emptyValue = Optional.empty<Any>()
                 expect(emptyValue).isEmptyFun()
             }
 
-            it ("throws an AssertionError") {
+            it("throws an AssertionError") {
```

Comment:
That's not your fault and I must have overlooked it in the review when isEmpty was added. This belongs into an own `context("is present and 2")`. Please change that. Once you have done this, you can add your use case to this context and to the context above (no need to repeat this again). See ResultFeatureAssertionsSpec line 76, both functions isSucessFeature and isFailureFeature are handled in the same context.

Reply:
I'm not sure to understand this one. My understanding by looking at ResultFeatureAssertionsSpec is that each  function should have their own context. This one was to validate that isEmpty on a nonempty value throws an assertion error. I will remove the duplicate describeFun in my next push. Let me know if it makes more sense once they have their own context.

Reply:
the context is related to the subject of the assertion and not the function. This was done wrong here and is correct in ResultFeatureAssertionsSpec. Thus my suggestion to wrap the `it("throws an AssertionError") {` into an own `context("is present and 2")` (outside of the `is empty` context). Now that you have an `context("is Preent")` you can move this test down into it